hooks/hooks.go

```diff
@@ -60,7 +60,9 @@ const (
 	// kinds represent will be prefixed by the workload/container name; for example,
 	// "mycontainer-pebble-ready".
 
-	PebbleReady Kind = "pebble-ready"
+	PebbleChangeUpdate Kind = "pebble-change-update"
+	PebbleClient       Kind = "pebble-client"
```

Comment:
what is mycontainer-pebble-client?

Reply:
It's a Pebble notice of type "client" (confusing name, I know). Basically a custom notice sent using `pebble notify` or the API.

Reply:
Renamed to `pebble-change-updated` (note the **d**) and `pebble-client-notice` for consistency and clarity.

Reply:
Per spec review discussion, we've settled on `pebble-change-updated` and `pebble-custom-notice`.